src/Raven.Server/Web/RequestHandler.cs

```diff
@@ -531,6 +531,18 @@ protected string GetQueryStringValueAndAssertIfSingleAndNotEmpty(string name)
             return values[0];
         }
 
+        protected string GetQueryOptionalStringValue(string name)
```

Comment:
what is wrong with GetStringQueryString? also can be removed if we will move he collections to ReverRevisionsRequest

Reply:
This method was removed